alembic/versions/f355b5458f82_bug_1474037_firefoxreality.py

```diff
@@ -0,0 +1,34 @@
+"""bug 1474037: add FirefoxReality
+
+Revision ID: f355b5458f82
+Revises: fab476fc2f5e
+Create Date: 2018-08-29 13:51:49.319243
+
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'f355b5458f82'
+down_revision = 'fab476fc2f5e'
+
+
+PRODUCTNAME = 'FirefoxReality'
+SORT = 4
```

Comment:
I decided to try this out so it's easier to copy/paste migrations in the future. I thought about doing a function in `socorro.lib`, but those change over time and the migrations that use them don't, so I think that's a bad model to follow.

Anyhow, just trying this out to see what happens.